vcd/resource_vcd_vapp_nat_rules.go

```diff
@@ -54,6 +54,12 @@ func resourceVcdVappNetworkNatRules() *schema.Resource {
 				ForceNew:    true,
 				Description: "vApp network identifier",
 			},
+			"enabled": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "Enable or disable NAT service.",
```

Comment:
Should state the default in description too.

Reply:
added